### ISSUE TITLE
Route naming is scoped improperly with Engines

### ISSUE THREAD

**Issue by trentmwillis** [ember-animation/liquid-fire#526]
Within an Engine, it looks like "route scoping" isn't properly accounted for. As an example, having a transition from `index -> guide`, where `guide` is an Engine's mount point, actually gets treated as `index -> application`. This is obviously unintuitive and likely not intended behavior.

You can see an example/reproduction here: https://github.com/ember-engines/ember-engines.com/pull/5#pullrequestreview-10828803

**Comment by ef4** [comment]
Agreed, we need to design how engines are reflected in the new RouteInfo related APIs from RFC 95. That's what liquid-fire will use (really, already uses, just in the private API version of them).

**Comment by buschtoens** [comment]
So, RFC 95 landed and was implemented some time ago. Is there a way I can help push this forward?

I was trying to use a `trantions.js` map *inside* my engine and that did not seem to work.

**Comment by buschtoens** [comment]
Oh, actually, it *did* work. I probably only forgot to restart my ember server—ember-engines still has issues when adding a new file.
